00-HTML-CSS-basics/HTML&CSS/basichtml.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8" />
+		<title>My todo list</title>
+		<h1>Hello World!</h1>
```

Comment:
The <head> tag shouldn't contain any visual markup, <h1> should go to the <body>